src/modules/core/components/Fields/Toggle/Toggle.tsx

```diff
@@ -18,7 +19,7 @@ interface Appearance {
 interface Props {
   appearance?: Appearance;
   name: string;
-  label?: string;
+  label?: string | MessageDescriptor;
   disabled?: boolean;
   tooltipText?: string;
```

Comment:
Same for tooltip text, should be a message descriptor and there should also be a `tooltipTextValues` prop.

_(Yes, I know you didn't add them like this, but while you are at it, might as well make the change)_